ae_reverse_proxy.gemspec

```diff
@@ -1,21 +1,23 @@
 # frozen_string_literal: true
 
-require_relative 'lib/ae_reverse_proxy'
+require_relative 'lib/ae_reverse_proxy/version'
 
 Gem::Specification.new do |spec|
-  spec.name                          = 'ae_reverse_proxy'
-  spec.version                       = AeReverseProxy::VERSION
-  spec.platform                      = Gem::Platform::RUBY
-  spec.authors                       = ['Appfolio']
-  spec.email                         = ['opensource@appfolio.com']
-  spec.summary                       = 'Gem for reverse proxying requests.'
-  spec.description                   = spec.summary
-  spec.license                       = 'MIT'
-  spec.required_ruby_version         = Gem::Requirement.new('>= 2.4.0')
-  spec.files                         = Dir['**/*'].select { |f| f[%r{^(lib/|Gemfile$|Rakefile|README.md|.*gemspec)}] }
-  spec.require_paths                 = ['lib']
+  spec.name                  = 'ae_reverse_proxy'
+  spec.version               = AeReverseProxy::VERSION
+  spec.platform              = Gem::Platform::RUBY
+  spec.author                = 'AppFolio'
+  spec.email                 = 'opensource@appfolio.com'
+  spec.description           = 'Gem for reverse proxying requests.'
+  spec.summary               = spec.description
+  spec.homepage              = 'https://github.com/appfolio/ae_reverse_proxy'
+  spec.license               = 'MIT'
```

Comment:
it looked like we removed the 'MIT' name from the license in a previous commit - does that change what we put here?

Reply:
I have been copying / pasting the standard MIT license into all these gems so they are consistent. I don't think we need that line that was in this gem as the license itself is the MIT license by definition.